server/src/main/java/org/apache/iotdb/db/mpp/execution/operator/process/RawDataAggregationOperator.java

```diff
@@ -42,6 +44,10 @@
  */
 public class RawDataAggregationOperator extends SingleInputAggregationOperator {
 
+  private final IWindowManager windowManager;
```

Comment:
I think IWindowManager should be replaced with IWindow, we can talk about it later in this afternoon.